sections/futures/MarketInfoBox/MarketInfoBox.tsx

```diff
@@ -19,20 +25,82 @@ const MarketInfoBox: React.FC<MarketInfoBoxProps> = ({
 	buyingPower,
 	marginUsage,
 	isMarketClosed,
+	potentialTrade,
+	maxLeverageValue,
+	currencyKey,
 }) => {
+	const potentialTradeDetails = useGetFuturesPotentialTradeDetails(
+		currencyKey as CurrencyKey,
+		potentialTrade
+	);
+
+	const previewTrade = potentialTradeDetails.data ?? null;
+
+	const previewTradeData = React.useMemo(() => {
+		const size = wei(potentialTrade?.size || zeroBN);
+		const potentialAvailableMargin =
+			previewTrade?.margin?.sub(previewTrade?.minInitialMargin) ?? zeroBN;
+		const potentialBuyingPower = previewTrade?.margin?.mul(maxLeverageValue) ?? zeroBN;
```

Comment:
This is incorrect because the `maxLeverageValue` is dependent on the position, not the actual max leverage of 10x. See comment above.